libs/fundamental-redirects/index.js

```diff
@@ -1101,13 +1130,46 @@ for (const [pattern, path] of [
   );
 }
 
+const LOCALE_ALIAS_PATTERNS = [
+  redirect(/^en\/(?<suffix>.*)$/i, ({ suffix }) => `/en-US/${suffix}`, {
+    permanent: true,
+  }),
+  redirect(
+    /^(?:cn|zh_cn|zh-hans|zh)\/(?<suffix>.*)$/i,
+    ({ suffix }) => `/zh-CN/${suffix}`,
+    {
+      permanent: true,
+    }
+  ),
+  redirect(
+    /^(?:zh_tw|zh-hant)\/(?<suffix>.*)$/i,
+    ({ suffix }) => `/zh-TW/${suffix}`,
+    {
+      permanent: true,
+    }
+  ),
+  redirect(/^pt\/(?<suffix>.*)$/i, ({ suffix }) => `/pt-PT/${suffix}`, {
+    permanent: true,
+  }),
+  ...localesWithoutCountryCodes.map((loc) => {
+    return redirect(
+      new RegExp(`^${loc}-${loc}\/(?<suffix>.*)$`, "i"),
```

Comment:
```suggestion
      new RegExp(`^${loc}-\\w{2}\/(?<suffix>.*)$`, "i"),
```
this makes `en-de` → `de` which I guess is applicable for all of these.